test/integration/dns_test.go

```diff
@@ -47,45 +44,36 @@ func TestDNS(t *testing.T) {
 		close(stop)
 	}, 50*time.Millisecond, stop)
 
-	// verify recursive DNS lookup is visible
-	m1 := &dns.Msg{
-		MsgHdr:   dns.MsgHdr{Id: dns.Id(), RecursionDesired: true},
-		Question: []dns.Question{{"foo.kubernetes.default.local.", dns.TypeA, dns.ClassINET}},
-	}
-	in, err := dns.Exchange(m1, server)
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	if len(in.Answer) != 1 {
-		t.Fatalf("unexpected response: %#v", in)
+	//verify recursive DNS lookup visible
+	tests := []struct {
+		dnsQuestionName string
+	}{
+		{
+			dnsQuestionName: "foo.kubernetes.default.local.",
+		},
+		{
+			dnsQuestionName: "www.google.com.",
```

Comment:
These aren't the same test.  www.google.com is recursive (testing that we forward resolution), foo.kubernetes.default.local is not.